nri/index.html

```diff
@@ -399,103 +408,114 @@ <h2>企業情報</h2>
     </div>
 
     <div class="footer-main-content">
-      <div>
+      <div class="footer-content-row">
         <ul>
-          <li><a href="#">ナレッジ・インサイト</a></li>
-          <li><a href="#">刊行物</a></li>
-          <li><a href="#">書籍</a></li>
-          <li><a href="#">コラム</a></li>
-          <li><a href="#">セミナー資料</a></li>
-          <li><a href="#">レポート</a></li>
-          <li><a href="#">用語解説</a></li>
+          <li><a href="#" class="content1">ナレッジ・インサイト</a></li>
```

Comment:
`content1`のようなクラス名を付与しないようにしましょう。

理由
- 1の意味がわからない

ここは太字にしたいから別のクラスを付与しているのだと思います。
こんな感じかな

```suggestion
          <li><a href="#" class="content-lead-title">ナレッジ・インサイト</a></li>
```

とか

```suggestion
          <li><a href="#" class="content-header">ナレッジ・インサイト</a></li>
```

とか

Reply:
content-lead-titleに変更対応しました。ありがとうございます！